rlberry/tests/test_agent.py

```diff
@@ -46,7 +46,7 @@ def feature_map_fn(_env):
 CONTINUOUS_STATE_AGENTS = [
     agents.RSUCBVIAgent,
     agents.RSKernelUCBVIAgent,
-    # torch_agents.DQNAgent,  # For now, DQN does not work with the generic check.
+    torch_agents.DQNAgent,
```

Comment:
Now that DQNAgent pass the checkes, you can remove the custom check I made line 66 as it has become useless.